style.css

```diff
@@ -1,7 +1,89 @@
 html {
-  font-size: 10px;
+  font-size: 12px;
 }
-title {
-  background-color: #3E3C3C;
-  color: blue;
+
+@media only screen and (max-width: 700px) {
```

Comment:
- [ ] Kindly check your code please. It seems like your CSS isn't loading.
![sss](https://user-images.githubusercontent.com/94551484/192070773-fdd64f9c-5989-4cf1-a35b-c30b71428fde.png)